lib/facilities/ppms/v1/client.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require_relative 'configuration'
+require_relative 'response'
+
+module Facilities
+  module PPMS
+    module V1
+      # Core class responsible for api interface operations
+      # Web forum and documentation (latest version of the ICD) located at:
+      # https://vaww.oed.portal.va.gov/pm/iehr/vista_evolution/RA/CCP_PPMS/PPMS_DWS/SitePages/Home.aspx
+      # Dev swagger site for testing endpoints
+      # https://dev.dws.ppms.va.gov/swagger
+      class Client < Common::Client::Base
+        configuration Facilities::PPMS::V0::Configuration
```

Comment:
I think you want the V1 configuration here Facilities::PPMS::V1::Configuration

Reply:
fixed that last night. forgot to push it up